aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -1,155 +1,36 @@
 
-Given the code snippet from the previous section and the explanation on t-shirt sizing for dimensions, if you were to implement the following code, you would have a Box with width of "384px" and height of "192px".
-
-```
-<Box width="medium" height="small" />
-// width = 384px, height = 192px
-```
-
-<Box width="large">
-  <Image src="/box-sizing.png" alt="Demonstration of t-shirt sizing on Box" />
-</Box>
-
-## T-shirt sizing for spacing and other styles
-
-In addition to overall component dimensions, t-shirt sizing can also be used to define spacing and other styling such as pad, gap, margin, and border.
-
-To create the pixel values for these styles, Grommet divides and/or multiplies the base unit of `24px` by various integers.
-
-Pad, gap, and margin are controlled by `edgeSize` and the corresponding pixel values also come from [`base.js`](https://github.com/grommet/grommet/blob/master/src/js/themes/base.js) in Grommet
-
-```
-// base.js in Grommet
-
-edgeSize: {
-    none: '0px',
-    hair: '1px', // for Chart
-    xxsmall: `${baseSpacing / 8}px`, // 3
-    xsmall: `${baseSpacing / 4}px`, // 6
-    small: `${baseSpacing / 2}px`, // 12
-    medium: `${baseSpacing}px`, // 24
-    large: `${baseSpacing * 2}px`, // 48
-    xlarge: `${baseSpacing * 4}px`, // 96
-
-}
-```
-
-Given this code snippet, if you were to implement the following code, you would have a Box with width of 384px, height of 192px, and pad of 48px.
-
-```
-<Box
-    width="medium"
-    height="small"
-    pad="large"
-/>
-```
-
-<Box width="medium" margin={{bottom: 'large'}}>
-  <Image
-    src="/edge-sizing.png"
-    alt="Demonstration of t-shirt sizing for pad on Box"
-  />
-</Box>
-
-Border sizing is controlled by `borderSize`.
-
-```
-// base.js in Grommet
-
-borderSize: {
-    xsmall: '1px',
-    small: '2px',
-    medium: `${baseSpacing / 6}px`, // 4
-    large: `${baseSpacing / 2}px`, // 12
-    xlarge: `${baseSpacing}px`, // 24
-}
-```
-
-Given this code snippet, if you were to implement the following code, you would have a Box with width of 384px, height of 192px, and a border with a width of 2px.
-
-```
-<Box 
-    border={{ color: 'border', size: 'small' }}
-    width="medium"
-    height="small" 
-/>
-```
-
-<Box width="medium" margin={{bottom: 'large'}}>
-  <Image
-    src="/border-sizing.png"
-    alt="Demonstration of t-shirt sizing for border on Box"
-  />
-</Box>
-
-### Composability and scaling between t-shirt sizes
-
-Sizes are intended to help give a UI a unified, harmonious experience. Therefore, there is a relationship between the pixel values that each size value maps to.
+- A need to consult with your designer about why a custom value is needed.
+- Something in the implementation can and should be simplified.
+- There could be a gap in the Design System which should be reported and addressed.
```

Comment:
```suggestion
- There could be a discrepancy in the Design System which should be reported and addressed.
```

since `gap` is one of our props maybe we can use another word